src/main/java/fi/hsl/common/transitdata/TransitdataProperties.java

```diff
@@ -84,4 +84,8 @@ public static long currentTimestamp() {
         OffsetDateTime utc = OffsetDateTime.now(ZoneOffset.UTC);
         return utc.toEpochSecond();
     }
+
+    public static String formatJoreId(String route, String direction, String startDate, String startTime) {
+        return REDIS_PREFIX_JORE_ID + "-" + route + "-" + direction + "-" + startDate + "-" + startTime;
```

Comment:
first '-' is probably not needed? 

```suggestion
        return REDIS_PREFIX_JORE_ID + route + "-" + direction + "-" + startDate + "-" + startTime;
```